lib/matplotlib/artist.py

```diff
@@ -302,10 +302,14 @@ def is_transform_set(self):
 
     def set_transform(self, t):
         """
-        Set the :class:`~matplotlib.transforms.Transform` instance
-        used by this artist.
+        Set the artist transform.
 
-        ACCEPTS: :class:`~matplotlib.transforms.Transform` instance
+        Parameters
+        ----------
+        t : `~.Transform`
+
+        ..
```

Comment:
This looks funny when rendered.  Is that what was wanted?  From your comment I assumed this wouldn't render...

<img width="676" alt="matplotlib_artist_artist_set_transform_ _matplotlib_2_0_2_post5339_g8cc0c57_documentation" src="https://user-images.githubusercontent.com/1562854/32456296-f9c0a56a-c2d9-11e7-97c1-06198d6aee42.png">

Reply:
I had to put it before the table, or indented within it (actually that will probably look better).  Fix coming...